test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_create.cpp

```diff
@@ -115,7 +115,8 @@ struct CreateCommandBufferRepeatedProperties : public BasicCommandBufferTest
             TEST_FAIL);
 
         cl_command_buffer_properties_khr invalid_properties[3] = {
-            CL_COMMAND_BUFFER_FLAGS_KHR, CL_INVALID_PROPERTY, 0
+            CL_COMMAND_BUFFER_FLAGS_KHR,
+            (cl_command_buffer_properties_khr)CL_INVALID_PROPERTY, 0
         };
```

Comment:
Not sure CL_INVALID_PROPERTY can be used in a property list like this. The spec only specifies its usage as a return value.

Reply:
I have used `-1` instead.